ldcbench.api/src/main/java/org/dice_research/ldcbench/rdf/SimpleTripleCreator.java

```diff
@@ -38,9 +39,23 @@ public SimpleTripleCreator(int baseGraphId, String[] resourceUriTemplates, Strin
 
     @Override
     public Triple createTriple(int sourceId, int propertyId, int targetId, int targetExtId, int targetExtGraphId) {
+        return createTriple(sourceId, propertyId, targetId, targetExtId, targetExtGraphId, false, false);
+    }
+
+    @Override
+    public Triple createTriple(int sourceId, int propertyId, int targetId, int targetExtId,
+            int targetExtGraphId, boolean withBlankNode, boolean withLiteral) {
```

Comment:
With the enum described above, the two last parameters could be replaced with `RDFNodeType objectType`.